vllm/README.md

```diff
@@ -2,6 +2,8 @@
 
 vLLM is a fast and easy-to-use library for LLM inference and serving. It has evolved into a community-driven project with contributions from both academia and industry. Intel, as one of the community contributors, is working actively to bring satisfying performance with vLLM on Intel® platforms, including Intel® Xeon® Scalable Processors, Intel® discrete GPUs, as well as Intel® Gaud® AI accelerators. This blog focuses on Intel® discrete GPUs at this time and brings you the necessary information to get the workloads running well on your Intel® graphics cards.
 
+The vLLM included in the latest docker image is 0.8.0.
```

Comment:
The vLLM used in the latest docker image is based on [v0.8.0](https://github.com/vllm-project/vllm/tree/v0.8.0).